pipelines/introduction-to-generic-pipelines/README.md

```diff
@@ -42,7 +42,9 @@ In this introductory tutorial you will learn how to create a generic pipeline an
 ### Setup
 
 This tutorial uses the `introduction to generic pipelines` sample from the https://github.com/elyra-ai/examples GitHub repository.
-1. Launch JupyterLab.
+1. Launch JupyterLab.    
+   > **Note**: When you start JupyterLab using the `jupyter lab` command, it loads the contents of the current working directory.  
+   > For examples, we recommend starting JupyterLab from a new directory on your system that is not an existing git repository. This will make sure you can clone the repository as mentioned in the next step.
```

Comment:
Typo:

```suggestion
   > For example, we recommend starting JupyterLab from a new directory on your system that is not an existing git repository. This will make sure you can clone the repository as mentioned in the next step.
```

Reply:
Do you think it is a good idea to replace it with - For `tutorials` instead of For `examples` ? The context here is to create a new directory when the user wants to try out the Elyra tutorials.

Reply:
We should not, because it is likely that many users will use github to share/backup their local JupyterLab artifacts (notebooks etc), not just those going through the tutorial.

Reply:
okay sounds good! Thanks 👍